src/main/java/net/openhft/chronicle/testframework/internal/BootstrapUtils.java

```diff
@@ -18,6 +19,10 @@
 import static java.lang.String.format;
 import static java.util.Arrays.asList;
 
+/**
+ * @deprecated To be replaced by {@link CodeStructureVerifier}.
+ */
+@Deprecated(/* to be removed in x.28 and replaced with BootstrapTestRunner */)
```

Comment:
I would argue that we just delete this in this PR, and not go through the deprecation cycle

Reply:
Good idea - done